postman/Manage Subscriber.postman_collection.json

```diff
@@ -0,0 +1,281 @@
+{
+	"info": {
+		"_postman_id": "cf8049f8-9671-4c2b-81a9-b9fbd088d029",
+		"name": "Manage Subscriber",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [{
+			"name": "Get Account",
+			"event": [{
+				"listen": "test",
+				"script": {
+					"id": "74e7dce1-167a-4675-8431-bdf2223fcd25",
+					"exec": [
+						"pm.test(\"Check consumer_key is defined and not empty\", function(){",
+						"   pm.expect(pm.environment.has(\"consumer_key\")).to.equal(true) && pm.expect(pm.environment.get(\"consumer_key\")).not.eql('');",
+						"});",
+						"",
+						"pm.test(\"Check consumer_secret is defined and not empty\", function(){",
+						"   pm.expect(pm.environment.has(\"consumer_secret\")).to.eql(true) && pm.expect(pm.environment.get(\"consumer_secret\")).not.eql('');",
+						"});",
+						"",
+						"pm.test(\"Check access_token is defined and not empty\", function(){",
+						"   pm.expect(pm.environment.has(\"access_token\")).to.eql(true) && pm.expect(pm.environment.get(\"access_token\")).not.eql('');",
+						"});",
+						"",
+						"pm.test(\"Check token_secret is defined and not empty\", function(){",
+						"   pm.expect(pm.environment.has(\"token_secret\")).to.eql(true) && pm.expect(pm.environment.get(\"token_secret\")).not.eql('');",
+						"});",
+						"",
+						"pm.test(\"Successfully get an account Id\", function(){",
+						"    pm.response.to.have.status(200);",
+						"    var jsonData = pm.response.json();",
+						"    pm.expect(jsonData.entries.length).to.be.above(0);",
+						"    pm.globals.set(\"accountId\",jsonData.entries[0].id ) ;   ",
+						"});",
+						"",
+						"pm.test(\"Successfully submitted request\", function() {",
+						"    pm.response.to.have.status(200);",
+						"});",
+						"",
+						""
+					],
+					"type": "text/javascript"
+				}
+			}],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_url}}/accounts",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"accounts"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Lists",
+			"event": [{
+				"listen": "test",
+				"script": {
+					"id": "f6163948-14a0-48e5-ad18-ee37ab1a3c0d",
+					"exec": [
+						"pm.test(\"Successfully get a  random list\", function() {",
+						"    pm.response.to.have.status(200);",
+						"});",
+						"",
+						"pm.test(\"At least two lists found\", function() {",
```

Comment:
1 list looks like it should be enough for this test

Reply:
Updated